examples/argocd/zarf.yaml

```
@@ -0,0 +1,51 @@
+kind: ZarfPackageConfig
+metadata:
+  name: argocd
+  description: Example showcasing installing ArgoCD
+
+components:
+  - name: argocd-helm-chart
+    required: true
+    charts:
+      - name: argo-cd
+        version: 5.38.1
+        namespace: argocd
+        url: https://argoproj.github.io/argo-helm
+        releaseName: argocd-baseline
+        valuesFiles:
+          - baseline/values.yaml
+    images:
+      - quay.io/argoproj/argocd:v2.7.7
+      - ghcr.io/dexidp/dex:v2.37.0
+      - public.ecr.aws/docker/library/redis:7.0.11-alpine
+  - name: argocd-apps
+    required: false
```

```suggestion
    required: true
```
Can we make this a required component in this example?